alcs-frontend/src/app/features/application/application.component.ts

```diff
@@ -128,17 +165,21 @@ export class ApplicationComponent implements OnInit, OnDestroy {
   destroy = new Subject<void>();
   childRoutes = appChildRoutes;
   unsubmittedRoutes = unsubmittedRoutes;
+  submittedLfngRoutes = submittedLfngRoutes;
 
   fileNumber?: string;
   application: ApplicationDto | undefined;
   reconsiderations: ApplicationReconsiderationDto[] = [];
   modifications: ApplicationModificationDto[] = [];
+  submission?: ApplicationSubmissionDto;
 
   isApplicantSubmission = false;
-  isSubmitted = false;
+  isSubmittedAlc = false;
+  isSubmittedLfng = false;
```

Comment:
I recommend renaming this to isPendingLfngReview or something similar, as currently it would imply its true if it was submitted to the LFNG but its actually true when its in a status where the lfng needs to do something.

Reply:
I initially named it something like that, but then this view is also true when Lfng returns the application and the applicant needs to do something. Any other recommendation for a name?